other/limit-hostpath-type-pv/limit_hostpath_type_pv.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: limit-hostpath-type-pv
+  annotations:
+    policies.kyverno.io/title: Limit hostPath PersistentVolumes to Specific Directories
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/minversion: 1.6.0
```

Comment:
I don't think the minimum version applies here so this annotation can probably be removed.

Reply:
I wasn't sure of that. I just removed that part. Thanks.